src/server.c

```diff
@@ -337,41 +262,30 @@ main(int argc, char **argv) {
                 strncpy(ca_path, optarg, sizeof(ca_path) - 1);
                 ca_path[sizeof(ca_path) - 1] = '\0';
                 break;
+            case ':':
             case '?':
-                break;
-            case 't':
-                optind--;
-                for (; optind < start && *argv[optind] != '-'; optind++) {
-                    char *option = strdup(optarg);
-                    char *key = strsep(&option, "=");
-                    if (key == NULL) {
-                        fprintf(stderr, "ttyd: invalid client option: %s, format: key=value\n", optarg);
-                        return -1;
-                    }
-                    char *value = strsep(&option, "=");
-                    free(option);
-                    struct json_object *obj = json_tokener_parse(value);
-                    json_object_object_add(client_prefs, key, obj != NULL ? obj : json_object_new_string(value));
-                }
-                break;
             default:
                 print_help();
                 return -1;
         }
     }
-    server->prefs_json = strdup(json_object_to_json_string(client_prefs));
```

Comment:
Personally, I don't think changing this option to accept a json string makes sense, since the current behavior works well, and this introduces a breaking change.